doc/usermanual/darkroom/modules/tone/filmicrgb.xml

```diff
@@ -0,0 +1,619 @@
+        </para></listitem>
+
+        <listitem><para>
+          If you plan using filmic rgb's chrominance preservation mode, avoid using any tone
+          mapping module as well as the base curve module. These may produce unpredictable color
+          shifts that would make the chrominance preservation useless. Neither of these modules
+          is usually needed if you use filmic rgb.
+        </para></listitem>
+
+      </itemizedlist>
+    </para>
+
+  </sect4>
+
+  <sect4>
+
+    <title>Usage</title>
+
+    <para>
+      The filmic rgb module aims at mapping the photographed scene dynamic range to the
+      (smaller) image dynamic range, and finally to the (even smaller) screen dynamic range.
+      This mapping is defined in three steps, each handled in a separate tab in the interface:
+      <itemizedlist>
+
+        <listitem><para>
+          The
+          <guilabel>scene</guilabel>
+          tab contains the <quote>input</quote> settings of the scene: what constitutes middle
+          grey, white and black in the photographed scene.
+        </para></listitem>
+
+        <listitem><para>
+          The
+          <guilabel>look</guilabel>
+          tab contains the parameters of the mapping applied to the input parameters defined in
+          the
+          <guilabel>scene</guilabel>
+          tab. Notably, this part of the module applies an S-shaped parametric curve to enhance
+          the contrast and remap the grey value to the middle grey of the display. This is
+          similar to what the base curve or tone curve modules do.
+        </para></listitem>
+
+        <listitem><para>
+          The
+          <guilabel>display</guilabel>
+          tab defined the output settings to map the transformed image to the display. In
+          typical use cases, this tab should only be used very rarely.
+        </para></listitem>
+
+      </itemizedlist>
+    </para>
+
+    <para>
+      The sliders' ranges of filmic rgb are limited to usual and safe values, but values are
+      allowed out of these ranges by clicking on the sliders with the right button and inputting
+      values on the keyboard. Filmic rgb has no neutral parameters resulting in a no-operation:
+      as soon as the module is enabled, the image is always at least slightly affected.
+    </para>
+
+    <para>
+      The curves at the top of the module are read-only and serve as a guide for the operations
+      performed on the sliders. The bright curve is the tone mapping curve, where the abscissa
+      represents the scene exposure, and the ordinate represents the display exposure. The dark
+      curve is the desaturation curve, representing the percentage of saturation as function of
+      the scene exposure.
+    </para>
+
+    <sect5>
+      <title>Middle-grey luminance (<guilabel>Scene</guilabel> tab)</title>
+<!-- IHU - okay, I'm not clear on color spaces. My understanding is that filmic
```

Comment:
Both filmics work in RGB. The luminance is the Y channel from XYZ space (Y represents the sensitivity of the "green" cone cells in the retina), the conversion from RGB to Y is simply `Y = a * R + b * G + c * B`, with a, b, c some constants depending on the working RGB space.